packages/widget.vite-plugin/src/build-plugin/__tests__/buildWidgetSetManifest.test.ts

```diff
@@ -20,7 +20,7 @@ import { buildWidgetSetManifest } from "../buildWidgetSetManifest.js";
 import type { WidgetBuildOutputs } from "../getWidgetBuildOutputs.js";
 
 describe("buildWidgetSetManifest", () => {
-  const WIDGET_SET_RID = "ri.foundry.main.widget-set.test-widget-set";
+  const WIDGET_SET_RID = "ri.widgetregistry.widget-set..test-widget-set";
```

Comment:
This is the correct format sorry `ri.widgetregistry..widget-set.test-widget-set`